.github/workflows/linkcheck.yml

```diff
@@ -0,0 +1,23 @@
+name: Link checker
+
+on:
+  pull_request:
+    # On every pull request
+  schedule:
+    - cron: '00 0 * * 1' # Every monday
+
+jobs:
+  link-check:
+    runs-on: ubuntu-latest
+    name: Link checker
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: gaurav-nelson/github-action-markdown-link-check@v1
```

Comment:
Note) I am waiting https://github.com/gaurav-nelson/github-action-markdown-link-check/pull/168 to be released
Otherwise, the GH action checks not only files in `docs` but also in `module/docs/mdoc/~` which contains vairables in links.